src/components/CorporateCampaign/LoanSearch/LoanSearchFilters.vue

```diff
@@ -37,13 +37,31 @@
 			</div>
 		</div>
 
-		<div v-if="filterChips.length" class="loan-filters__chips">
-			<kv-chip
-				v-for="(filter, index) in filterChips"
-				:key="index"
-				:title="cleanChipName(filter.name)"
-				@click-chip="handleRemoveFilter(filter)"
-			/>
+		<div
+			v-if="filterChips.length"
+			class="loan-filters__chips-wrapper"
+		>
+			<div class="row">
+				<div
+					class="loan-filters__chips small-12 large-9 xxlarge-10 columns"
+					:class="{'loan-filters__chips--collapsed' : chipsCollapsed}"
+				>
+					<kv-chip
+						v-for="(filter, index) in filterChips"
+						:key="`chip-${index}`"
+						:title="cleanChipName(filter.name)"
+						@click-chip="handleRemoveFilter(filter)"
+					/>
+				</div>
+				<div class="small-12 large-3 xxlarge-2 columns">
+					<kv-button
+						class="loan-filters__chips-toggle text-link"
+						@click.native="chipsCollapsed = !chipsCollapsed"
+					>
+						{{ chipsCollapsed ? `Show all ${filterChips.length} filters` : 'Hide filters' }}
+					</kv-button>
+				</div>
```

Comment:
Could you please drop in a "Reset All" button here too.

The action needs to emit `this.$emit('reset-loan-filters');` and then the parent will need to listen for `@reset-loan-filters="handleResetLoanFilters"`